### PR TITLE
Universal/docs/post processing docs

### PR DESCRIPTION
**Pull request by Katstodian**
PR for initial post-processing docs for Universal 19.3:

- new landing page for PP (removed mention of PPv2)
- Updated ToC to include PP elements
- updated Asset page with PP 
- Updated comparison page with OnRenderImage = Supported
- Added Effect pages for Panini Projection, Lens Distortion, Channel Mixer, Color Adjustments, Color Curves, Lift Gamma and Gain, Shadow Midtones Highlights, Split Toning, White Balance, and Film Grain.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/TableOfContents.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
-* [About UniversalRP](index.md)
-* [Getting started with UniversalRP](getting-started-with-universalrp.md)
-  * [Creating a new Project with UniversalRP](creating-a-new-project-with-universalrp.md)
-  * [Upgrading your existing Project to use UniversalRP](installing-universalrp-into-an-existing-project.md)
-  * [Configuring UniversalRP for use](configuring-universalrp-for-use.md)
-* [Inside UniversalRP](inside-universalrp.md)
-  * [The UniversalRP Asset](universalrp-asset.md)
+* [About Universa lRP](index.md)
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/TableOfContents.md]: typo

File: com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ This table provides an overview of the current features supported in the Univers
 | OnPreCull                                                    | Yes                                                          | Not supported                                                |
 | OnPreRender                                                  | Yes                                                          | Not supported                                                |
 | OnPostRender                                                 | Yes                                                          | Not supported                                                |
-| OnRenderImage                                                | Yes                                                          | Not supported                                                |
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md]: I've made a mistake when copying the table entry for you! (sorry about that!). OnRenderImage is not supported, the one that was added is the one below (OnRenderObject)

Comment (Katstodian) [reply com.unity.render-pipelines.universal/Documentation~/universalrp-builtin-feature-comparison.md]: Fixed.

File: com.unity.render-pipelines.universal/Documentation~/installing-universalrp-into-an-existing-project.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
+Before you can start using Universal RP, you must configure it by creating a Scriptable Render Pipeline Asset and changing your Graphics settings. To learn how, see [Configuring Universal RP for use](configuring-universalrp-for-use.md).
 
-**Note:** Switching to UniversalRP in an existing Project consumes a lot of time and resources. UniversalRP uses custom lit shaders and is not compatible with the Built-in Unity lit shaders. You will have to manually change or convert many elements. Instead, consider [starting a new Project with UniversalRP](creating-a-new-project-with-universalrp.md).
+**Note:** Switching to Universal RP in an existing Project consumes a lot of time and resources. Universal RP uses custom lit shaders and is not compatible with the Built-in Unity lit shaders. You will have to manually change or convert many elements. Instead, consider [starting a new Project with Universal RP](creating-a-new-project-with-universalrp.md).
 
+**Note:** The Universal Render Pipeline uses its own [post-processing](integration-with-post-processing.md). If you have the post-processing version 2 package installed in your Project already, you need to delete it before you upgrade.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/installing-universalrp-into-an-existing-project.md]: Double checking: @Chman, having ppv2 there when upgrading should not break the project, but it's indeed recommended to delete it.

**Comment by phi-lira** [comment]
Adding @Chman for coverage. However seems good.

**Comment by Katstodian** [comment]
@phi-lira, @Chman has already approved all pp content in the docs process. (apart from the comparison page comment about Universal using a native resolition)

**Comment by Katstodian** [comment]
@Verasl - definitely! After the second batch of PP Docs?

**Pull request merged**